aws-cpp-sdk-s3/include/aws/s3/S3Errors.h

```diff
@@ -54,12 +54,13 @@ enum class AWS_S3_API S3Errors
   NO_SUCH_KEY,
   NO_SUCH_UPLOAD,
   OBJECT_ALREADY_IN_ACTIVE_TIER,
-  OBJECT_NOT_IN_ACTIVE_TIER
+  OBJECT_NOT_IN_ACTIVE_TIER,
+  SLOWDOWN
```

Comment:
This value is autogenerated from the s3 model, can you post the exact string you recieved from S3 so I can properly update their model. Then this change will go out automatically.